### DISCUSSION TITLE
Qz companion app on Garmin no pwr

### DISCUSSION THREAD

**Discussion by dstrimble** [cagnulein/qdomyos-zwift#2854] [Q&A]
Everything seems to be connected, heart rate is showing on the qz phone app, but no pwr is related to the watch. Garmin Instinct 2.
Also when the app saves the activity to Garmin connect it shows as running instead of biking. Can that be adjusted?

**Comment by cagnulein** [comment]
@dstrimble in order to send power and all the metrics to garmin you need to run qz on android with ant+ or Mac silicon. Where are you running qz?

**Reply by dstrimble** [reply]
Google pixel 9

**Reply by dstrimble** [reply]
If this means I need an ant + dongle then I suppose I don't really need the qz companion Garmin app. With the dongle I just add as accessories and use indoor cycling activity.

**Comment by cagnulein** [comment]
Exactly you don't need the companion at all. Qz will send directly the
metrics over ant+

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno dom 1 dic 2024 alle 15:46 Daniel Trimble ***@***.***>
ha scritto:

**Reply by dstrimble** [reply]
Ok. Thank you. I was hoping for a way around needing the ant dongle.

**Comment by cagnulein** [comment]
Unfortunately it's a garmin limitation

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno dom 1 dic 2024 alle 15:52 Daniel Trimble ***@***.***>
ha scritto:

**Reply by dstrimble** [reply]
You mentioned Mac, does this mean iPads have built in ant+? I haven't tried running qz on ipad

**Comment by cagnulein** [comment]
Nope via Mac, the Bluetooth is not private and the garmin works. No iPad or
iPhone are compatible

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno dom 1 dic 2024 alle 16:42 Daniel Trimble ***@***.***>
ha scritto:

**Reply by dstrimble** [reply]
I have a MacBook Air I can try. Do you have documentation on that?

**Comment by cagnulein** [comment]
It's a silicon one? Simple download qz from the app store and enable the
garmin compatibility setting and you will find the sensor on your watch

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno dom 1 dic 2024 alle 16:52 Daniel Trimble ***@***.***>
ha scritto:

**Reply by dstrimble** [reply]
Dang, it's Intel. Thanks for the info

**Comment by cagnulein** [comment]
;(

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno dom 1 dic 2024 alle 17:15 Daniel Trimble ***@***.***>
ha scritto:

**Reply by NaimNaimov** [reply]
Hi @cagnulein, 

I have the app on a MacBook Pro M4, and try to send the info as a ANT+ data, to my Garmin Venu Sq 2. 

The thing is I am not certain that the app is connecting to the bike (it's Orion Force 700, which is using FTMS connection).
Could it be that it's not supported or I am doing something wrong?

---

When I turn the app on, it simply states that it has found the nearby bluetooth devices.
I have the Garmin options for ANT+ enabled.

Can you please help?

**Comment by cagnulein** [comment]
on mac you can't use ant+, use bluetooth. enable the garmin bluetooth
setting under accessories the garmin options in qz

let me know

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno mer 3 set 2025 alle 21:57 Naim Naimov ***@***.***>
ha scritto:

**Comment by devdems** [comment]
Why do we need ant+?. I have Garmin Fenix watch and can connect via bluetooth to sensor like HRM. Why can it get bluetooth sensor from the phone and the data that qz app is providing which is running on that phone?

**Reply by cagnulein** [reply]
read this https://robertoviola.cloud/2023/03/24/why-my-sensor-cant-connect-to-a-garmin-device/

**Reply by devdems** [reply]
@cagnulein so basically i need to add ant+ dongle to android and it should be working. Do you have info if this works on my phone which is Google Pixel 9?

**Reply by cagnulein** [reply]
i have a google pixel 8a and it's working. So you should be fine. I'm not using anymore ant+ since i'm using the garmin connect feature right now

**Reply by devdems** [reply]
Tnx. I just got this feature after signing for beta program. This will work also for my needs.
